runtime/src/bank/tests.rs

```diff
@@ -1115,6 +1115,123 @@ fn test_distribute_rent_to_validators_overflow() {
     }
 }
 
+#[test]
+fn test_distribute_rent_to_validators_rent_paying() {
+    solana_logger::setup();
+
+    const RENT: u64 = 55;
+    const TOTAL_RENT: u64 = RENT * 2;
+
+    let rent_paying_validator = ValidatorVoteKeypairs::new_rand();
+    let rent_exempt_validator = ValidatorVoteKeypairs::new_rand();
+    let keypairs = vec![&rent_paying_validator, &rent_exempt_validator];
+    let genesis_config_info = create_genesis_config_with_vote_accounts(
+        sol_to_lamports(1000.),
+        &keypairs,
+        vec![sol_to_lamports(1000.); 2],
+    );
+    let mut genesis_config = genesis_config_info.genesis_config;
+    genesis_config.rent = Rent::default(); // Ensure rent is non-zero, as genesis_utils sets Rent::free by default
+
+    let bank = Bank::new_for_tests(&genesis_config);
+
+    // Make one validator almost rent-exempt, less RENT
```

Comment:
Can we add another test case to cover where pre-rent account balance is 0?

Reply:
Done